tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -39,6 +39,8 @@
   "port": {{ ELASTICSEARCH_PORT }},
 }]
 
+CONTACT_MAILING_ADDRESS: {{ PLATFORM_NAME }} - {% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}
```

Comment:
Wrong copy-paste! :-D This should be written as:

```
CONTACT_MAILING_ADDRESS = "..."
```

Reply:
@BbrSofiane will you have time to fix this? After that we should be ready to merge.

Reply:
Fixed! I forgot to push the change.